.circleci/e2e-job-ignore-patterns.txt

```diff
@@ -0,0 +1,10 @@
+.md
+.jpg
```

Comment:
If you're doing any screenshot comparison tests (which are kind of a pain), then some of these formats might need to be included. Likewise, if an image's dimensions change, it could make a button inaccessible, etc. So this may be a little strict.

Reply:
Good point. Screenshot comparison testing is on the longer term road map.